src/Microsoft.AspNet.Identity/LoggerExtensions.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Identity
+{
+    public static class LoggerExtensions
```

Comment:
@suhasj When you go back to instance methods on UserManager, RoleManager and SignInManager, take a look at getting rid of this level of helpers. Seem largely unnecessary. You can do the check on the other ones and I don't think it is that much duplication.